src/sentry/lang/javascript/processing.py

```diff
@@ -83,6 +83,30 @@ def sourcemap_images_from_data(data):
     return get_path(data, "debug_meta", "images", default=(), filter=is_sourcemap_image)
 
 
+def map_symbolicator_process_js_errors(errors):
+    if errors is None:
+        return []
+
+    mapped_errors = []
+
+    for error in errors:
+        if error["type"] == "invalid_abs_path":
+            mapped_errors.append({"type": EventError.JS_MISSING_SOURCE, "url": error["abs_path"]})
+        if error["type"] == "missing_sourcemap":
+            mapped_errors.append({"type": EventError.JS_MISSING_SOURCE, "url": error["abs_path"]})
+        elif error["type"] == "invalid_location":
+            mapped_errors.append(
+                {
+                    "type": EventError.JS_INVALID_SOURCEMAP_LOCATION,
+                    "column": error["column"],
+                    "row": error["row"],
```

Comment:
since you are mapping this manually anyway, it could have been the chance to use `line` :-P

Reply:
https://github.com/getsentry/symbolicator/pull/1116

Will need to restart CI once master image is published.